ngrinder-core/build.gradle

```diff
@@ -36,13 +34,6 @@ task makeAgentDependencies {
         def classpath = ""
         sourceSets.main.compileClasspath.each {
             def libName = it.getName()
-            if (libName == "grinder-3.9.1.jar") {
-                return
-            }
-            if (libName == "grinder-3.9.1-patch.jar") {
-                classpath += "grinder-3.9.1;"
-                return
-            }
             def removedVersion = libName.findAll(/(.*?)-\d+/).join("")
             if (removedVersion == "") {
                 return
```

Comment:
This could be simplify with code below (without substring call on adding classpath)
```groovy
def libNameWithoutVersion = libName.findAll(/(.*?)(?=-\d+)/).join("")
if (libNameWithoutVersion == "") {
    return
}
classpath += "${libNameWithoutVersion};"
```